tests/runtime-new/rust/xcrate/test.wit

```diff
@@ -0,0 +1,75 @@
+//@ tests = ['intermediate', 'leaf']
```

Comment:
does this mean `runner` is run with both `intermediate` and `leaf`?

Reply:
Good question, and mostly points to this being a bad name. I've renamed this to `dependencies` which should make it more clear that these are dependencies of the `runner` component, not separate and individual tests. The sum total of all three components are what makes a single test.